Dummy patch to serve as marker separating patches for mutt-nntp

--- /dev/null
+++ b/.mutt.patched
@@ -0,0 +1 @@
+# dummy patch to serve as marker separating patches for mutt-nntp
